### ISSUE TITLE
confused by loss function forward params

### ISSUE THREAD

**Issue by cswwp** [qubvel/segmentation_models.pytorch#48]
Hi qubvel, i am confused by the i was confused by loss function forward params.

As show in utils/functions.py, the def iou and f_score calculate IOU loss and DICE loss with pr and gt as params, and does it means pr (torch.Tensor) is a tensor with shape[batch, channel, width, height]? but why the annotation is  "A list of predicted elements" as follows, i'm confused with what is the pr, tensor,  with shape[batch, channel, width, height] or list with element num batch * channel * width * height, which one?

**Comment by qubvel** [comment]
This should be just a tensor (batch, channels, height, width) not list, will fix the description, thanks.

**Comment by cswwp** [comment]
> This should be just a tensor (batch, channels, height, width) not list, will fix the description, thanks.

ok

**Comment by cswwp** [comment]
> This should be just a tensor (batch, channels, height, width) not list, will fix the description, thanks.

This's a great base framework, thanks very much!
